java/lang/security/audit/bad-hexa-conversion.yaml

```diff
@@ -1,7 +1,10 @@
 rules:
 - id: bad-hexa-conversion
+  metadata:
+    cwe: "CWE-704: Incorrect Type Conversion or Cast"
+    owasp: "A3: Sensitive Data Exposure"
```

Comment:
Maybe it is A6: Security Misconfiguration

Reply:
This one is hard to map, but I went with A3 because this isn't really a "configuration" issue. It weakens the crypto, so I binned it with other crypto checks. 🤷‍♂️